### DISCUSSION TITLE
w3m-button

### DISCUSSION THREAD

**Discussion by TornGuard** [WalletConnect/walletconnect-monorepo#5389] [Developer Support] [answered]
How can i add custom styles to w3m-button?

**Comment by rtomas** [comment, marked as answer]
You can use this hook to open an create your custom button.
https://docs.reown.com/appkit/react/core/hooks#useappkit

**Reply by MaximusDAO** [reply]
Are there other optional properties for the appkit button than those shown on the docs? I'm using vanilla JavaScript and would like to make it not automatically reconnect if the user had already connected. Some type of "autoConnect=false" thing. Or if there's another way to prevent automatic connection that would be great to know.

**Comment by TornGuard** [comment]
Great this worked. Is there anyway to change colour of that spining svg which circulates around selected wallet when connect wallet is pending?

**Reply by rtomas** [reply]
No, you can raise a feature request in the AppKit repository.
